### PR TITLE
Set up testing

### PR DESCRIPTION
**Pull request by pkruithof**
I followed the Symfony instructions for testing, and just used the phpunit bridge, which manages the phpunit dependency itself. Apparently it installs it in the `bin` directory.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,24 +1,6 @@
...
-
 ###> symfony/framework-bundle ###
 APP_ENV=dev
 APP_SECRET=fa19261df587444abec550dd06929194
 #TRUSTED_PROXIES=127.0.0.0/8,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16
 #TRUSTED_HOSTS='^localhost|example\.com$'
 ###< symfony/framework-bundle ###
-
-## echo -n "GOOGLE_SERVICE_ACCOUNT=$(cat ~/google-account-credentials.json | base64 -w0)" | tee -a .env.local
-GOOGLE_SERVICE_ACCOUNT=
```
Comment (pkruithof) [inline .env]: Using personal accounts is preferred over having a service account

File: docker-compose.yaml [hunk 2]
```diff
@@ -1,21 +1,14 @@
...
-    context: docker/php
-    target: php-cli
-  volumes:
-    - ./:/app:delegated
-
 services:
   php:
-    <<: *php
-
-  composer:
```
Comment (pkruithof) [inline docker-compose.yaml]: For development it makes no sense to have separate php/composer services.

File: tests/Infrastructure/FirestoreRepositoryTest.php [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+        putenv("FIRESTORE_EMULATOR_HOST={$emulator}");
+
+        $firestore = new FirestoreClient();
+
+        $this->repo = MockRepository::create($firestore);
+    }
+
+    /** @test */
+    public function it_can_save_and_load_an_aggregate_root(): void
```
Comment (pkruithof) [inline tests/Infrastructure/FirestoreRepositoryTest.php]: Test copied from newly builds

File: README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+```shell script
+docker-compose build
+docker-compose run --rm php composer install
+```
+
+Authenticate Google Cloud:
+
+```shell script
+gcloud auth login
+gcloud auth application-default login
```
Comment (jvdlaar) [inline README.md]: How does this work on a k8s pod?

Comment (pkruithof) [reply README.md]: With a service account, you can set that with an environment variable that gets picked up automatically by the Firestore client.

**Pull request merged**